fhir-model/src/main/java/com/ibm/fhir/model/util/FHIRUtil.java

```diff
@@ -171,6 +173,13 @@
         UsageContext.class,
         Dosage.class));
     private static final Map<String, String> CONCRETE_TYPE_NAME_MAP = buildConcreteTypeNameMap();
+    private static final OperationOutcome ALL_OK = OperationOutcome.builder()
+                                                            .issue(Issue.builder()
+                                                                    .severity(IssueSeverity.INFORMATION)
```

Comment:
is there an alternative class to put this ALL_OK. outcome in?

Reply:
FHIRUtil was the spot with all the buildOperationOutcome helpes, so it made the most sense.  If those ever get split out, this should get split out with them.